src/components/composer/composer/composer.ts

```diff
@@ -289,19 +286,14 @@ export class Composer extends Component<CellComposerProps, SpreadsheetChildEnv>
 
   setup() {
     this.DOMFocusableElementStore = useStore(DOMFocusableElementStore);
-    this.autoCompleteState = useLocalStore(AutoCompleteStore);
     onMounted(() => {
       const el = this.composerRef.el!;
       if (this.props.isDefaultFocus) {
         this.DOMFocusableElementStore.setFocusableElement(el);
       }
       this.contentHelper.updateEl(el);
     });
-    this.env.model.selection.observe(this, {
-      handleEvent: () => this.autoCompleteState.hide(),
-    });
     onWillUnmount(() => {
-      this.env.model.selection.detachObserver(this);
```

Comment:
can we get rid of the detachObserver then ? :D

Reply:
yes sir :)
`detachObserver` is now gone